GitUI/Editor/Diff/AnsiEscapeUtilities.cs

```diff
@@ -0,0 +1,468 @@
+﻿using System.Diagnostics;
```

Comment:
Was this file copied from somewhere? If so, we need to provide the attribution.

Reply:
No, this was handwritten from specifications in wikipedia and Git source. Attribution to the debug coloring, but no shared code.

Reply:
My apologies for assuming you copied this from somewhere. 
This is some elaborate piece, well done! You know that I will be asking for some tests, right? 😆

Reply:
> My apologies for assuming you copied this from somewhere.
This is some elaborate piece, well done! You know that I will be asking for some tests, right? 😆

Should I by the way be insulted or flattered by this comment? (I choose the latter...)